README.md

```diff
@@ -2,23 +2,6 @@
 [![Releases](https://img.shields.io/badge/version-v0.0.1-orange.svg)](https://sigs.k8s.io/cluster-api-provider-ibmcloud/releases "Cluster API provider IBM Cloud latest release")
 [![Go Report Card](https://goreportcard.com/badge/sigs.k8s.io/cluster-api-provider-ibmcloud)](https://goreportcard.com/report/sigs.k8s.io/cluster-api-provider-ibmcloud)
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
DO we still want to keep the toc? Why remove it?

Reply:
we hardly have much content here in the readme to maintain the index, wdys?